packages/components/stories/Progressbar.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Progressbar from '../src/components/progressbar';
+
+const Template = (args) => <Progressbar {...args} />;
+
+export const Default = Template.bind({});
+
+export const ProgressBar: React.FC = () => (
+  <>
+    <h2 className="tk-mt-4h">Determinate (linear)</h2>
+    <div>
+      <Progressbar className="tk-ml-h" value={'100'} progress={'determinate'} />
```

Comment:
Here you could create an example with a timeout where you update the value of the ProgressBar and display as well the numeric percentage.